test/IntegrationTests/SmokeTests.cs

```diff
@@ -191,6 +192,22 @@ public async Task PrometheusExporter()
         }
     }
 
+#if !NETFRAMEWORK
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public async Task SubmitLogs()
+    {
+        using var collector = await MockLogsCollector.Start(Output);
+        collector.Expect(logRecord => Convert.ToString(logRecord.Body) == "{ \"stringValue\": \"Example log message\" }");
+
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_LOGS_PARSE_STATE_VALUES", "true");
```

Comment:
This is needed for the tests to pass. Once the other PR gets merged will remove parse state environment variable.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1300/commits/ea6fd539335c4607e462fae0d7ee7db247e085a9